<!-- 
  Launch file for sensor pre-processing
  - Nodes for pointcloud processing
-->
<launch>

  <!-- Sensor pre-processing -->
  <node pkg="height_mapping" type="sensor_processor_node" name="sensor_processor" output="screen" />


</launch>